_layouts/default.html

```diff
@@ -206,7 +207,7 @@
 
                 carouselContainer.slick({
                   arrows: false,
-                  autoplay: true,
+                  autoplay: false,
```

Comment:
I don't think I meant to include that. Autoplay true should be OK for now, because I am working on the carousel styling in my own pre-JS version. Will share with you soon.